.github/workflows/lint-test.yaml

```diff
@@ -5,6 +5,17 @@ on: pull_request
 jobs:
   lint-test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Latest k8s versions. There's no series-based tag, nor is there a latest tag.
+        k8s:
+          - 1.16.15
+          - 1.17.17
+          - 1.18.15
+          - 1.19.7
+          - 1.20.2
+          - 1.21.1
+          #- 1.22.0
```

Comment:
The commit message includes 1.22.
This line excludes 1.22.
Is this a typo or on purpose?

Reply:
This PR is not ready yet, wait for Velero v1.6.3 release.